react/features/overlay/components/AbstractReloadableOverlay.js

```diff
@@ -0,0 +1,20 @@
+/* global APP */
+import AbstractOverlay from './AbstractOverlay';
+
+/**
+ * Implements an abstract React Component for overlay with ability to
+ * reload the page.
+ */
+export default class AbstractReloadableOverlay extends AbstractOverlay {
```

Comment:
I'm not sure it's worth it to make an abstract component here, since all it's shared is a single one-line function which will need to be refactored anyway.  I'd remove this.